paimon-common/src/main/java/org/apache/paimon/lookup/hash/HashLookupStoreReader.java

```diff
@@ -87,6 +94,15 @@ public class HashLookupStoreReader
             int keyLengthCount = dataInputStream.readInt();
             // Max key length
             int maxKeyLength = dataInputStream.readInt();
+            // BloomFilter
+            bloomFilterEnabled = dataInputStream.readBoolean();
+            if (bloomFilterEnabled) {
+                this.bloomFilter =
```

Comment:
Can this BF into cache?

We have an implementation in Flink: `HashTableBloomFilter`.

BF can base on memory segment too.

Reply:
I'll take a look